src/app/helptext/system/ntpservers.ts

```diff
@@ -38,17 +38,20 @@ export const helptext_system_ntpservers = {
     minpoll: {
       placeholder: T("Min. Poll"),
       tooltip: T(
-        "Power of 2 in seconds; cannot be lower than 4 or\
- higher than <i>Max. Poll</i>."
+        "Calculated in powers of 2. Enter the exponent to be \
```

Comment:
This is still pretty confusing.  Let's fix the label of the field while here, nobody abbreviates "min" or "max" with a period. The description should first tell what this field is, then describe the calculation.  Suggestion:
`The minimum polling interval, in seconds, as a power of 2. For example, <i>6</i> means 2^6, or 64 seconds. The default is 6, minimum value is 4.`